integration-cli/docker_cli_links_test.go

```diff
@@ -141,22 +117,15 @@ func (s *DockerSuite) TestLinksHostsFilesInject(c *check.C) {
 	out, _ = dockerCmd(c, "run", "-itd", "--name", "two", "--link", "one:onetwo", "busybox", "top")
 	idTwo := strings.TrimSpace(out)
 
-	c.Assert(waitRun(idTwo), check.IsNil)
+	c.Assert(waitRun(idTwo), checker.IsNil)
 
 	contentOne, err := readContainerFileWithExec(idOne, "/etc/hosts")
-	if err != nil {
-		c.Fatal(err, string(contentOne))
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("contentOne: %s", string(contentOne)))
 
 	contentTwo, err := readContainerFileWithExec(idTwo, "/etc/hosts")
-	if err != nil {
-		c.Fatal(err, string(contentTwo))
-	}
-
-	if !strings.Contains(string(contentTwo), "onetwo") {
-		c.Fatal("Host is not present in updated hosts file", string(contentTwo))
-	}
-
+	c.Assert(err, checker.IsNil, check.Commentf("contentTwo: %s", string(contentTwo)))
+	// Host is not present in updated hosts file
+	c.Assert(string(contentTwo), checker.Contains, "onetwo")
```

Comment:
Yep yep, I forgot that it handled the comment above too :sweat_smile:. My bad :wink: